package.json

```diff
@@ -48,6 +48,7 @@
     "3id-resolver": "^0.0.5",
     "@babel/runtime": "^7.4.5",
     "did-jwt": "^0.1.3",
+    "eth-sig-util": "^2.2.0",
```

Comment:
Not sure if we need to optimize for this now, but this library seems quite heavy for just verifying a signature.